fix(perf): Add missing docs for missing web vitals chart (#3021)
getsentry/sentry-docs: src/docs/product/performance/transaction-summary.mdx

This graph uses a modified version of [ASAP](https://arxiv.org/pdf/1703.00983.pdf) smoothing. The Trends graph shows the unsmoothed data of the Display data, along with a smoothed version of that data.

### Web Vitals

> This graph shows the P75 of [First Paint](/product/performance/web-vitals/#first-paint-fp), [First Contentul Paint](/product/performance/web-vitals/#first-contentful-paint-fcp), [Largest Contentful Paint](/product/performance/web-vitals/#largest-contentful-paint-lcp), and [First Input Delay](/product/performance/web-vitals/#first-input-delay-fid) over time. This view can help you understand the web vitals associated with the transaction is doing over time.

Reviewer: ```suggestion
This graph displays the [P75 threshold](/product/performance/metrics/#p75-threshold) of [First Paint](/product/performance/web-vitals/#first-paint-fp), [First Contentful Paint](/product/performance/web-vitals/#first-contentful-paint-fcp), [Largest Contentful Paint](/product/performance/web-vitals/#largest-contentful-paint-lcp), and [First Input Delay](/product/performance/web-vitals/#first-input-delay-fid) over time. This view helps you understand how the web vitals associated with the transaction are performing over time.
```

Author: Thank you! 😄
